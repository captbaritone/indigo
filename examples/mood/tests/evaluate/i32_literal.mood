pub fn test(): i32 {
  10_i32
}