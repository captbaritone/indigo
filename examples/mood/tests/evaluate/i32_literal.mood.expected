-----------------
INPUT
----------------- 
pub fn test(): i32 {
  10_i32
}
-----------------
OUTPUT
-----------------
10